CHANGELOG.md

```diff
@@ -231,7 +233,7 @@ Skierfe is a mountain in the Sarek national park, and the inspiration for the lo
 | `bwa`                  | 0.7.17      | unchanged   |
 | `cancerit-allelecount` | 4.0.2       | 4.3.0       |
 | `cnvkit`               | 0.9.6       | 0.9.9       |
-| `control-freec`        | 11.6        | 11.6        |
+| `control-freec`        | 11.6        | unchanged   |
```

Comment:
never realised you put in the list 🤯 ❤️  should the header be 3.0.2?

Reply:
Did that with #752
And no, because actually this is what modules were update in 3.0 (no modules updates (or no version updated to be exact) in `3.0.1` or `3.0.2`)

Reply:
ok makes sense